README.rst

```diff
@@ -1 +1,86 @@
-The api for the master facility list.
\ No newline at end of file
+==========
+Master Facility List API
+==========
+.. image:: https://circleci.com/gh/MasterFacilityList/mfl_api.svg?style=shield
+    :target: https://circleci.com/gh/MasterFacilityList/mfl_api
+
+This is the backend for the Master Facility list. 
+It provides an api through which one is able to access various resources provided by master facility list; Including facilities and their services.
```

Comment:
Grammar; "API" is usually capitalized; "Master Facility List" is a compound noun - use sentence case. The first "I" after the semicolon should be lower-case ( that is not a sentence boundary ).